management/clipper_manager.py

```diff
@@ -85,39 +85,110 @@ class Clipper:
     """
 
     def __init__(self, host, user, key_path):
-        env.key_filename = key_path
-        env.user = user
-        env.host_string = host
         self.host = host
+        env.host_string = host
+        if not self.host_is_local():
+            env.user = user
+            env.key_filename = key_path
         # Make sure docker is running on cluster
+        self.start_docker_if_necessary()
+
+    def host_is_local(self):
```

Comment:
Since these are private methods can you prefix them with _. E.g. `host_is_local` to `_host_is_local`